### PR TITLE
2.2.0 - PayFrame

### PR DESCRIPTION
**Pull request by enesf**
## 2.2.0 - 2015-07-21
* add ssl support for the payment form page
* Integration of PayFrame to enable use of SAQ A for easier PCI DSS 3.0 compliance
* transaction description initially set

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
Hi @enesf,

Can you please remove your folder nbprojects from the PR ? Theyare only used by NetBeans

File: components/models/configurationModel.php [hunk 1]
```diff
@@ -71,6 +71,27 @@ class ConfigurationModel {
...
 
+        /**
+	 * @var string
+	 */
+	private $pci;
+
+        /**
+         * @return string
+         */
+        function getPci()
```
Comment (Quetzacoalt91) [inline components/models/configurationModel.php]: The visibility is missing (public | private | protected)

File: components/models/configurationModel.php [hunk 2]
```diff
@@ -71,6 +71,27 @@ class ConfigurationModel {
...
+        {
+            return $this->pci;
+        }
+
+        /**
+         * @param string $pci
+         */
+        function setPci($pci)
```
Comment (Quetzacoalt91) [inline components/models/configurationModel.php]: The visibility is missing here too

File: pigmbhpaymill.php [hunk 3]
```diff
@@ -163,10 +163,11 @@ public function hookdisplayHeader()
 		if ($this->name === Tools::getValue('module'))
 		{
 			$this->context->controller->addCSS(__PS_BASE_URI__.'modules/pigmbhpaymill/css/paymill_styles.css');
+			$this->context->controller->addCSS('http://fonts.googleapis.com/css?family=Open+Sans:400,300,700');
```
Comment (Quetzacoalt91) [inline pigmbhpaymill.php]: This line throws an "Mixed active content" error when the shop is in HTTPS

**Comment by Harenkamp** [comment]
Hi @enesf , @Quetzacoalt91,

i've updated the mentioned todos. 

Beste regards
Stefan Harenkamp

**Comment by Quetzacoalt91** [comment]
Hi @Harenkamp,

Thanks, your PR has just been merged and the changes will be available from tomorrow.

Best regards

**Pull request merged**